UEBPrep/app/models/playlist.rb

```diff
@@ -1,5 +1,6 @@
 class Playlist < ActiveRecord::Base
   validates :name, presence: true
+  validate :user_id, presence: true
```

Comment:
Yes you can, but we shouldn't have object methods that could be nil a lot. This just ensures that any playlist in the database will have a user_id.